docs/src/constraints.md

```diff
@@ -876,6 +876,60 @@ julia> @constraint(model, M * y + q ⟂ y)
 [y[1] + 2 y[2] + 5, 3 y[1] + 4 y[2] + 6, y[1], y[2]] ∈ MathOptInterface.Complements(2)
 ```
 
+## Special Ordered Sets (SOS1 and SOS2)
+
+A Special Ordered Set (SOS) is an ordered set of variables with particular characteristics.
+
+If a vector `x` is in a Special Ordered Set type 1 (SOS1), at most one element (or variable) of `x`
+can take a non-zero value, all others being zero, in a feasible solution. To induce ordering,
+`weights` can be provided; as such, they should be unique values. The `k`th element in the set
+corresponds to the `k`th weight in `weights`. 
```

Comment:
Maybe: 

If a vector of variables `x` is in a Special Ordered Set of Type I (SOS1), then at most one element of `x` can take a non-zero value, and all other elements must be zero. 

Although not required for feasibility, solvers can benefit from an ordering of the variables (e.g., the variables represent different factories to build, at most one factory can be built, and the factories can be ordered according to cost). To induce an ordering, `weights` can be provided; as such, they should be unique values. The `k`th element in the ordered set corresponds to the `k`th weight in `weights` when the weights are sorted.